exotica_python/src/Exotica.cpp

```diff
@@ -574,7 +574,7 @@ PYBIND11_MODULE(_pyexotica, module)
     kdlFrame.def("getAngleAxis", [](KDL::Frame* me) { return getRotationAsVector(*me, RotationType::ANGLE_AXIS); });
     kdlFrame.def("getQuaternion", [](KDL::Frame* me) { return getRotationAsVector(*me, RotationType::QUATERNION); });
     kdlFrame.def("getTranslation", [](KDL::Frame* me) { Eigen::Vector3d tmp; for (int i = 0; i < 3; i++) { tmp[i] = me->p.data[i]; } return tmp; });
-    kdlFrame.def("get", [](KDL::Frame* me) { return getFrame(*me); });
+    kdlFrame.def("getFrame", [](KDL::Frame* me) { return getFrame(*me); });
```

Comment:
Also, please ask Vlad about this change I am not sure it does what's intended in the first place.

Reply:
Have asked and he agrees on getFrame, will update elsewhere where this is being called (I'm assuming in exotica_example?), will do some searching.

Reply:
It is not being used as far as I can tell and that is what was my point - what is this function meant to do? You can already instantiate from another frame, you can also get all data with the other methods, and you can instantiate from Eigen::Vectors or rotation matrices? @VladimirIvan